packages/pluggableWidgets/color-picker-web/src/ColorPicker.editorConfig.ts

```diff
@@ -0,0 +1,5 @@
+import { Properties } from "@mendix/piw-utils-internal";
+
+export function getProperties(__values: any, defaultProperties: any): Properties {
+    return defaultProperties;
```

Comment:
Unless we don't plan on supporting this in Studio, it's good to include a `transformGroupsIntoTabs` like we do with all widgets.

Reply:
Disregard previous comment, if I add `transformGroupsIntoTabs ` `getProperties` will make sense